.dockerignore

```diff
@@ -1 +1,4 @@
 **/node_modules
+*.egg-info
+Dockerfile
+*.Dockerfile
```

Comment:
Leaving dockerfiles out allows the build cache to work properly when you modify Dockerfile.